pkgs/applications/science/biology/snpeff/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "snpeff-${version}";
-  version = "4.3p";
+  version = "4.3T";
```

Comment:
`snpeff --help` refers to itself as `snpEff version SnpEff 4.3t (build 2017-11-24 10:18), by Pablo Cingolani`, so maybe you should keep it lowercase?